### PR TITLE
Install Go Meta Linter dependencies

### PR DESCRIPTION
**Pull request by szkl**
Signed-off-by: Sonmez Kartal <sonmez@koding.com>

### CODE REVIEW

File: scripts/gometalinter.sh [hunk 1]
```diff
@@ -2,9 +2,12 @@
 
 set -o errexit
 
-export GOPATH=${GOPATH:-$(pwd)/go} \
-       GOBIN=${GOBIN:-$GOPATH/bin}
+export GOPATH=${GOPATH:-$(pwd)/go}
+export GOBIN=${GOBIN:-$GOPATH/bin}
```
Comment (rjeczalik) [inline scripts/gometalinter.sh]: It's not needed, GOBIN defaults to $GOPATH/bin if not set.

Comment (szkl) [inline scripts/gometalinter.sh]: GOBIN is shorter than GOPATH/bin.

**Comment by szkl** [comment]
Updated.

File: scripts/gometalinter.sh (hunk 1 above)
Comment (rjeczalik) [inline scripts/gometalinter.sh]: I meant you can safely remove that export line.

**Review by rjeczalik — APPROVED** [review]

**Pull request merged**